scripts/test_aktualizr_deb_and_update.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+set -e
+
+/persistent/selfupdate_server.py 8000&
+
+dpkg-deb -I /persistent/aktualizr.deb && dpkg -i /persistent/aktualizr.deb
+aktualizr --version | grep $(cat /persistent/aktualizr-version) && aktualizr-info
```

Comment:
Does this build a version of Aktualizr with the correct --version?

On my machine I tried the following:

~~~
# docker build -t advancedtelematic/aktualizr-noostree -f Dockerfile.noostree  .
# docker run -v /persistent:/persistent advancedtelematic/aktualizr-noostree src/scripts/build-ubuntu.sh
# dpkg -i /persistent/aktualizr.deb
# aktualizr --version
Current aktualizr version is:
~~~

Reply:
It should, but I found another issue, looks like after rebase dissapeared `-DBUILD_DEB=ON ` from src/scripts/build-ubuntu.sh.

Reply:
But CI logs shows that aktualizr built with debian support otherwise it should fail